### DISCUSSION TITLE
I'm very troubled that why can't I install this document

### DISCUSSION THREAD

**Discussion by zhangyancq** [dalcinl/igakit#7] [Q&A]
I don't know if it's because I have gfortran installed in msys2, or if there are other reasons why I can't install igakit.And there doesn't seem to be a setup.py file.

**Comment by dalcinl** [comment]
I've never tried to build igakit on a Windows machine using WSL, so it may not work. 
Indeed, there is no setup.py. igakit uses meson and meson-python as a build system, and that does not require the traditional setup.py file.

The usual way to install this project would be
```console
$ git clone https://github.com/dalcinl/igakit.git
$ python3 -m pip install ./igakit
```
If that does not work, you should show me the output, otherwise I cannot even guess.
